README.md

```diff
@@ -101,6 +102,17 @@ After library is included, call `expect()` method on global `UX` object passing
 
 Each individual zone configuration object contains of zone's `label` that will be used as a name of corresponding [W3C UserTiming API `measure`](https://www.w3.org/TR/user-timing/#performancemeasure) and `marks` array of individual event label strings that zone groups together, each individual label will be used when recording corresponding events as [W3C UserTiming API `mark`](https://www.w3.org/TR/user-timing/#performancemark).
 
+#### Setting Custom Handlers
+
+In cases when monitoring solution you use does not support W3C UserTiming API natively, you might want to provide a custom method of recording the results.
+
+You can do that by specifying custom handlers for `mark` and `measure` events which will read data from UserTiming API, format the results accordingly and pass them to your service.
+
+To accomplish that, you can call `UX.config()` method and pass it a configuration object. Following keys are currently supported:
```

Comment:
This might be a dumb question, but do you ever envision a situation where we'd want to have a different callback on a specific measure or mark event?

Reply:
Yes, we might, but I feel that main callback can do the filtering and routing to particular function.

From this library's perspective, it's just one point connection to alternative way of reporting numbers - the rest will be responsibility of specific implementation, e.g. our specific NewRelic integration.